backend/src/emcr/emcr.service.ts

```diff
@@ -204,6 +204,11 @@ export class EmcrService {
         query.page,
         query.status,
       );
+      personnel.forEach(person => {
+        if(person.personnel.recommitment){
+        person.personnel.recommitment = person?.personnel?.recommitment?.filter(itm => itm.program === Program.EMCR)
```

Comment:
This was necessary?

Reply:
Might not be the best way to handle this, but I thought we might not want the bcws data returned from the emcr endpoint, and vice versa.

Reply:
I could make the change to filter the entity in the same way as we do with the personnel entity (class transformer + program) but it would require updates everywhere that we're mapping the recommitment entities into the recommitment RO's. (might tech debt that though as it would be cleaner)

Reply:
Going to merge just to get it through testing - I have made a note to look at revising this.